README.md

```diff
@@ -389,3 +458,9 @@ Gets the linked affiliation and person information for an email address in JSON
 }
 ```
 
+## License
+
+This work is licensed under the [MIT license](https://mit-license.org/).
+
+**Note**: This work uses the [Spring Framework](https://github.com/spring-projects) and derivatives from the Spring framework family, which are licensed under [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
Needs to reference micronaut framwork :)

Reply:
thanks 👍 I overlooked that